src/dotnet/Telemetry.cs

```diff
@@ -9,18 +10,28 @@ namespace Microsoft.DotNet.Cli
 {
     public class Telemetry : ITelemetry
     {
+        public const double TimeoutInMilliseconds = 100;
+
         private bool _isInitialized = false;
         private TelemetryClient _client = null;
 
         private Dictionary<string, string> _commonProperties = null;
         private Dictionary<string, double> _commonMeasurements = null;
+        private Task _trackEventTask = null;
+
+        private const int ReciprocalSampleRateValue = 5;
```

Comment:
I guess it really comes down to: "What are we going to use this telemetry information for?"

---
In reply to: [61365252](https://github.com/dotnet/cli/pull/2724#discussion_r61365252) [](ancestors = 61365252)